jax/_src/scipy/special.py

```diff
@@ -1303,14 +1320,16 @@ def _expi_pos(x):
     [_expint1, _expint2, _expint3, _expint4, _expint5, _expint6, _expint7],
   )
 
+def _expi_neg(x: Array) -> Array:
```

Comment:
Made this an explicit function both for ease of typing, and for better jit cacheing